test/test.ts

```diff
@@ -33,6 +35,13 @@ describe('linkinator', () => {
     scope.done();
   });
 
+  it('should filter links with same URL', async () => {
```

Comment:
"Same URL" is not correct here.  This is making sure we ignore URL fragments in uniqueness of the url.  Maybe 'should ignore fragments when parsing DOM'

Reply:
thank you, done!